### PR TITLE
[17.0] [MIG] sale_delivery_split_date: migration to 17.0

### PR DESCRIPTION
**Pull request by siemenv**
Depends on:

- https://github.com/OCA/sale-workflow/pull/2915
- https://github.com/OCA/sale-workflow/pull/2962

### CODE REVIEW

**Comment by siemenv** [comment]
Pipelines are failing because of missing sale_order_line_date dependency which is also in pull request.

**Comment by rousseldenis** [comment]
> Pipelines are failing because of missing sale_order_line_date dependency which is also in pull request.

@siemenv Thanks for this. FYI, to help reviewers testing this on runboat and tests passing, you can embed the depending PR following : https://github.com/OCA/maintainer-tools/wiki/Use-temporary-reference%28s%29-to-another-pull-request%28s%29

**Comment by rousseldenis** [comment]
@siemenv Moreover, to help people reviewing too the depending PR, don't forget to mention it (with link) in this one description. Thanks

**Comment by niboo-ave** [comment]
Hello,
I am looking forward using this module in v17. Thank you for your work.

How long before this will be merged ?

**Comment by siemenv** [comment]
> Hello, I am looking forward using this module in v17. Thank you for your work.
> 
> How long before this will be merged ?

**Comment by niboo-rmu** [comment]
Hello,
I am looking forward using this module in v17. 
Would you have any idea on when this will be available ?

Thank you.

**Comment by rousseldenis** [comment]
@RomainMuschh @antoineVerlant OCA contributions depend on good will. So, you can review it (and dependencies) in order to get the work done. Thanks!

**Review by sanderlienaerts — APPROVED** [review]
Looks good to me. Note that there are some commits left to squash: https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

File: sale_delivery_split_date/models/sale_order_line.py
Comment (BT-etejeda) [inline sale_delivery_split_date/models/sale_order_line.py:34]: Hi @siemenv , I red the discussion in the migration to v14 but I'm not really convinced that it is right because the security lead time is not taking into account here. Could you or anyone else give an opinion about that? If I install this module as it is, the security days i have configured in my DB will be without effect.

**Review by BT-etejeda — COMMENTED** [review]
Apart from what I commented, everything is OK for me.

**Review by BT-etejeda — COMMENTED** [review]
@siemenv you can clean the test-requirements.txt now that the dependencies are merged

File: sale_delivery_split_date/models/sale_order_line.py
Comment (BT-etejeda) [reply sale_delivery_split_date/models/sale_order_line.py:34]: ping @siemenv

**Comment by HaraldPanten** [comment]
Closing this PR, as it has been superseded

**Pull request closed without merging**
